cli/azd/internal/appdetect/appdetect.go

```diff
@@ -326,7 +326,7 @@ func detectAny(ctx context.Context, detectors []projectDetector, path string, en
 			log.Printf("Found project %s at %s", project.Language, path)
 
 			// docker is an optional property of a project, and thus is different from other detectors
-			docker, err := detectDockerInDirectory(path, entries)
+			docker, err := detectDockerInDirectory(*project, entries)
```

Comment:
Why is this change necessary?

Reply:
necessary before, but not used now, updated.